src/main/kotlin/assimp/Importer.kt

```diff
@@ -191,6 +191,11 @@ constructor() {
             }
         }
 
+    var ioHandler: IOSystem
```

Comment:
Move this into base importer, allows for default function args

Reply:
This is the way the original assimp does it though
http://assimp.sourceforge.net/lib_html/class_assimp_1_1_importer.html#a1161f46318af18bb86dfe0fc3edea4df

Is there some Kotlin trickery I can use to mirror that into the importers ( probably has to do with var and get() = something something ) ?

Reply:
Actually, after playing around myself, you're good on this.